packages/zosjobs/src/GetJobs.ts

```diff
@@ -366,13 +367,14 @@ export class GetJobs {
      * @returns {Promise<string>} - promise that resolves to the spool content
      * @memberof GetJobs
      */
-    public static async getSpoolContentById(session: AbstractSession, jobname: string, jobid: string, spoolId: number) {
+    public static async getSpoolContentById(session: AbstractSession, jobname: string, jobid: string, spoolId: number, encoding?: string) {
```

Comment:
Should we move the params to a typed object? (in V3)

Reply:
I was thinking that any changes to interfaces and parameters would be a different story, since this was primarily meant to get the two branches reconciled. It might be nice to do that change on this API....